### PR TITLE
Add CountdownTimer fragment

### PR DESCRIPTION
**Pull request by Prabhutva**
The countdown timer to Quark 2019.

### CODE REVIEW

File: app/src/main/java/com/macbitsgoa/events/timer/TimerCardFragment.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    @Override
+    public View onCreateView(final LayoutInflater inflater,
+                             final ViewGroup container, final Bundle savedInstanceState) {
+        final MaterialCardView selfCard =
+                (MaterialCardView) inflater.inflate(R.layout.fragment_timer_card,
+                        container, false);
+        final TextView tv= (TextView) selfCard.findViewById(R.id.timer_text);
+
+        int daysToGo =1;
```
Comment (Rushi98) [inline app/src/main/java/com/macbitsgoa/events/timer/TimerCardFragment.java]: do not hardcode the values

use those defined in **app/build.gradle**, for example, here you can use the string `startDate`

File: app/src/main/java/com/macbitsgoa/events/timer/TimerCardFragment.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                int seconds = (int) (millis / 1000) % 60 ;
+                int minutes = (int) ((millis / (1000*60)) % 60);
+                int hours   = (int) ((millis / (1000*60*60)) % 24);
+                String text = String.format("%02d days, %02d hours, %02d minutes, %02d seconds",days,hours,minutes,seconds);
+                tv.setText(text);
+            }
+
+            @Override
+            public void onFinish() {
+                tv.setText("Kabooom");
```
Comment (Rushi98) [inline app/src/main/java/com/macbitsgoa/events/timer/TimerCardFragment.java]: no :smiling_imp:

File: app/src/main/java/com/macbitsgoa/events/timer/TimerCardFragment.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            }
+        }.start();
+        //selfCard.setOnClickListener(view ->
```
Comment (Rushi98) [inline app/src/main/java/com/macbitsgoa/events/timer/TimerCardFragment.java]: delete unnecessary code. no commenting outs

**Comment by Prabhutva** [comment]
It was a initial code, i'll remove the unnecessary elements while making
the ui😅
ᐧ

**Pull request merged**